addons/minio/2022-09-25T15-44-53Z/install.sh

```diff
@@ -190,6 +190,20 @@ function minio_migrate_from_rgw() {
 
     minio_wait_for_health
 
+    export DID_MIGRATE_ROOK_OBJECT_STORE=0
+    DID_MIGRATE_ROOK_OBJECT_STORE=$(kubectl -n kurl get configmap kurl-migration-from-rook -o jsonpath='{ .data.DID_MIGRATE_ROOK_OBJECT_STORE }')
+    if [ "$DID_MIGRATE_ROOK_OBJECT_STORE" == "1" ]; then
+        logWarn "Object store is set as migrated previously. Not migrating object store again."
+        return
+    fi
+
+    export DID_MIGRATE_ROOK_OBJECT_STORE=0
```

Comment:
i think you have the code duplicated here

Reply:
I removed that and I think I  pushed again.
 Thank for getting this one.